lib/lens/compression.rb

```diff
@@ -0,0 +1,97 @@
+require 'bson'
+require 'beefcake'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.